lib/auth/auth.js

```diff
@@ -100,7 +100,9 @@ function extractParams(request, log, awsService, data) {
  * @param {object} log - Logger object
  * @param {function} cb - the callback
  * @param {string} awsService - Aws service related
- * @param {RequestContext[]} requestContexts - array of RequestContext
+ * @param {RequestContext[] | null} requestContexts - array of
```

Comment:
Why allow a null value when we could simply provide an empty array and not modify the code ? I do not have the proper vision to understand the impact of this.